client/settings/settings-warnings/incompatibility-notice.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import interpolateComponents from '@automattic/interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import { Notice } from '@wordpress/components';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
+import './style.scss';
+
+const WooPayIncompatibilityNotice = () => (
+	<Notice
+		status="warning"
+		isDismissible={ false }
+		className="express-checkout__notice express-checkout__incompatibility-warning"
+	>
+		<span>
+			<NoticeOutlineIcon
+				style={ {
+					color: '#F0B849',
+					fill: 'currentColor',
+					marginBottom: '-5px',
+					marginRight: '10px',
+				} }
+				size={ 20 }
+			/>
+		</span>
+		<span>
+			{ __(
```

Comment:
We are missing text domain 'woocommerce-payments' as a second argument here.

Reply:
Fixed with 9d010d60035e392a504f252b45f43ff73e242ce6